fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBottomSheet.kt

```diff
@@ -262,43 +311,120 @@ private fun TranslationsDialogInfoMessage(
     }
 }
 
+@Suppress("MagicNumber")
 @Composable
 private fun TranslationsDropdown(
     header: String,
     modifier: Modifier = Modifier,
+    isInLandscapeMode: Boolean = false,
+    translateLanguages: List<Language>,
 ) {
-    Column(modifier, verticalArrangement = Arrangement.spacedBy(4.dp)) {
+    var expanded by remember { mutableStateOf(false) }
+
+    var selectedLanguage by remember {
+        mutableStateOf(
+            translateLanguages.last().localizedDisplayName,
+        )
+    }
+
+    val density = LocalDensity.current
```

Comment:
Maybe move this below or above all our remember {} variables so we can just group them all together.

Reply:
Done